### PR TITLE
Sadeq-db-week1

### PR DESCRIPTION
**Pull request by sadeq93**

### CODE REVIEW

File: Week1/Homework/ex1.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const mysql = require("mysql");
+const connections ={
+    host: "localhost",
+    user: "hyfuser",
+    password: "hyfpassword"
+  }
+const db = mysql.createConnection(connections);
+  db.connect((err) => {
```
Comment (unmeshvrije) [inline Week1/Homework/ex1.js:8]: indentation is odd here

File: Week1/Homework/ex1.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    if(err) throw err;
+    console.log('connecting ...')
+  });
+function createMySQL_Query (mySQL_Command){
```
Comment (unmeshvrije) [inline Week1/Homework/ex1.js:12]: I would name this function as `run_mysql_query`

**Review by unmeshvrije — COMMENTED** [review]
Overall looks good and works.
Just fix minor indentation and naming comments

File: Week1/Homework/ex1.js (hunk 1 above)
Comment (sadeq93) [reply Week1/Homework/ex1.js:8]: sorry my teacher I did not understand what "indentation is odd here" means actually, may you explain it more please

**Comment by unmeshvrije** [comment]
db.connect statement should start in the column 1 of the line. There are
unnecessary spaces before "db"

Also there are unnecessary spaces before the "}"

   Regards,
   Unmesh Joshi

**Pull request closed without merging**